src/main/java/cz/fi/muni/pa165/esports/dao/PlayerDaoImpl.java

```diff
@@ -0,0 +1,45 @@
+package cz.fi.muni.pa165.esports.dao;
+
+import cz.fi.muni.pa165.esports.entity.Player;
+import cz.fi.muni.pa165.esports.enums.Gender;
+import org.springframework.stereotype.Repository;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import java.util.List;
+
+/**
+ * @author Radovan Tomasik
+ */
+@Repository
+public class PlayerDaoImpl implements PlayerDao {
+    @PersistenceContext
+    private EntityManager em;
+
+    @Override
+    public Player findById(Long id) {
+        return em.find(Player.class, id);
+    }
+
+    @Override
+    public void create(Player player) {
+        em.persist(player);
+    }
+
```

Comment:
I believe we wanted to have a `remove()` DAO method for each entity as well? (+ in the interface)

Reply:
Done